src/__tests__/prompt.test.ts

```diff
@@ -0,0 +1,30 @@
+import { OCR_SYSTEM_PROMPT, RECEIPT_SCAN_PROMPT, RECEIPT_VALIDATION_PROMPT } from '../const/prompt';
```

Comment:
the idea is to grab all the prompts from the prompt-library tooling being developed as we speak